spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,9 +31,16 @@
     context "on #{plat[:name]} we should behave ourselves" do
       let(:platform) { plat }
       let(:cur_plat) { Vanagon::Platform::DSL.new(plat[:name]) }
+      let (:wixtestfiles) { "/tmp/spec/fixtures/wix/resources/windows/wix" }
 
       before do
         cur_plat.instance_eval(plat[:block])
+        FileUtils.mkdir_p(wixtestfiles)
+        FileUtils.cp_r("spec/fixtures/wix/resources/windows/wix", "#{wixtestfiles}/..")
+      end
+      after do
+        FileUtils.rm_rf("#{WORK_BASE}")
+        FileUtils.rm_rf("/tmp/spec")
       end
```

Comment:
Cool, I was thinking it was strange to have wix specific code executed outside of our wix specific `before` and `after` calls.